### ISSUE TITLE
Test DIAMOND database was built with older version of DIAMOND(v0.8.x)

### ISSUE THREAD

**Issue by kdews** [Lafond-LapalmeJ/MCSC_Decontamination#30]
Hello,

I am using MCSC to decontaminate a _de novo_ assembled transcriptome (thank you!), but ran into a small issue when attempting to run `test.ini` after install.

Conda environment:
`conda create -n mcsc diamond=0.9.22 r=3.1.2`

Attempt to run MCSC on `test.ini`:
`sh MCSC_decontamination.sh test/test.ini`
> Not using alignment correction
> Validating input file format...
> Input file format seems to be OK.
> Running DIAMOND blast...
> diamond v2.0.8.146 (C) Max Planck Society for the Advancement of Science
> Documentation, support and updates available at http://www.diamondsearch.org
> 
> #CPU threads: 12
> Scoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)
> Temporary directory: ./test
> #Target sequences to report alignments for: 25
> Opening the database...  [0.111s]
> **Error: Database was built with an older version of Diamond and is incompatible.**
> Extracting DIAMOND blast taxonomy...
> readline() on closed filehandle IN at /home1/kdeweese/bin/MCSC_Decontamination/scripts/diamond_to_tagc.pl line 34.
> Formating the DIAMOND output...
> Running the MCSC algorithm...
> Computing the White-Ratio (WR) index and evaluating the clusters...
> The phylum to keep is Chordata
> Top phylum in ./test/taxo_uniq.txt :
> Use of uninitialized value $taxa1 in concatenation (.) or string at /home1/kdeweese/bin/MCSC_Decontamination/scripts/cluster_eval.pl line 117.
> Use of uninitialized value $taxa2 in concatenation (.) or string at /home1/kdeweese/bin/MCSC_Decontamination/scripts/cluster_eval.pl line 117.
> Use of uninitialized value $taxa3 in concatenation (.) or string at /home1/kdeweese/bin/MCSC_Decontamination/scripts/cluster_eval.pl line 117.
> Use of uninitialized value $taxa4 in concatenation (.) or string at /home1/kdeweese/bin/MCSC_Decontamination/scripts/cluster_eval.pl line 117.
> Use of uninitialized value $taxa5 in concatenation (.) or string at /home1/kdeweese/bin/MCSC_Decontamination/scripts/cluster_eval.pl line 117.
> The phylum Chordata is not present in taxo_uniq.txt file. Aborting.

### TLDR, Error: Database was built with an older version of Diamond and is incompatible.

Output of:
`diamond dbinfo --db test.dmnd`
> diamond v0.9.22.123 | by Benjamin Buchfink <buchfink@gmail.com>
> Licensed under the GNU AGPL <https://www.gnu.org/licenses/agpl.txt>
> Check http://github.com/bbuchfink/diamond for updates.

> Database format version = 0
> Diamond build = 84
> Sequences = 11558
> Letters = 4941376

A database format version of zero [indicates](https://github.com/bbuchfink/diamond/wiki/5.-Advanced-topics) that the `test.dmnd` database was built with DIAMOND v0.8.x, not v0.9.22 as is required by MCSC.

**Could you possibly reupload the `test.dmnd` database built with DIAMOND v0.9.22?**

Thank you,
Kelly

**Comment by lukaszsobala** [comment]
You can use the uniref90 database made with your version of DIAMOND for the test data - the search might take a bit longer, however. I use diamond 2.0.9 with MCSC_Decontamination and it works.
